chart/values.yaml

```diff
@@ -69,11 +72,6 @@ es-data:
     master: "false"
     ingest: "false"
     data: "true"
-  volumeClaimTemplate:
```

Comment:
keep the `volumeClaimTemplate` template block, you will still need it for a real/production cluster. The `persistence.enabled` flag should take of using or not using `volumeClaimTemplate`.